spring-kafka/src/main/java/org/springframework/kafka/annotation/KafkaListenerAnnotationBeanPostProcessor.java

```diff
@@ -1070,4 +1089,14 @@ protected boolean isEmptyPayload(Object payload) {
 
 	}
 
+	/**
+	 * Post processes each annotation.
+	 *
+	 * @since 2.7.2
+	 *
+	 */
+	public interface AnnotationEnhancer extends BiFunction<KafkaListener, AnnotatedElement, KafkaListener> {
```

Comment:
OK. You have introduced a more specific type, but you don't mention it in the docs I've commented above.
Thanks